drivers/azure/azureutil/tenantid.go

```diff
@@ -41,3 +84,7 @@ func findTenantID(env azure.Environment, subscriptionID string) (string, error)
 	}
 	return m[1], nil
 }
+
+func saveTenantID(path string, tenantID string) error {
+	return ioutil.WriteFile(path, []byte(tenantID), 0600)
```

Comment:
makes sense, I forgot about this as our `saveToken` handles this already. I'll turn this into write to tempfile and move to here.